front/scripts/i18n-api-errors.ts

```diff
@@ -3,16 +3,28 @@ import i18next from 'i18next';
 import { noop } from 'lodash';
 import type { OpenAPI } from 'openapi-types';
 
+import i18nData from '../public/locales/fr/errors.json' assert { type: 'json' };
+
 // relative the project's root
 const openapi_path = '../editoast/openapi.yaml';
 // relative to this file
 const i18n_error_path = '../public/locales/fr/errors.json';
 
+type OpenAPIErrorSchema = {
```

Comment:
Instead of defining our own type here, can't we use something from `openapi-types`? Would be better to runtime assert that the OpenAPI data conforms to the shape we assume, rather than cast.